gradle/solr/force-versions.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+// Force slf4j version so that it does not get auto promoted from 1x to 2-alpha due to Jetty dependency.
```

Comment:
The technique here is new to me.  I see enforcedPlatform documented [here](https://docs.gradle.org/current/userguide/platforms.html#sub:bom_import).

Okay I guess.  It seems like this is a little-known solution to the problem we are solving.

Reply:
So I know it was added a LONG time ago by dweiss during the initial gradle migration:
* https://github.com/apache/solr/commit/ca8661bc3ae4cf5076f5df6412f04f78350e238c
* https://github.com/apache/solr/commit/73e8b49f0d077e4c85d3d5c726d1acc0c6d2ac69

It was removed in SOLR-15987 
 since we were able to get to a consistent slf4j version again: https://github.com/apache/solr/commit/d9373b418474d28676e3812cf42e33d7b7f51c51

Reply:
Addressed in 14b141e3a9f2842f7173d14ba05a8a625cb46bd2 - this fixes the slf4j logging stuff too you ran into. more details in separate comment in this PR.

Reply:
Details in https://github.com/apache/solr/pull/585#issuecomment-1288324958 about slf4j-api issue.